data/find-questions.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * To use this:
+ * - Modify `predicateCallback` to look for questions that
+ *   match your requirements
+ * - run: node find-questions.js
+ */
+
+/* eslint-disable import/no-commonjs */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const fs = require("fs");
+const Path = require("path");
+
+// ==========================
+// MODIFY THIS WHEN SEARCHING
+// ==========================
+
+/**
+ * predicateCallback used to check if the given question matches
+ * the rules you're looking for
+ *
+ * @param {PerseusRenderer} q the question to check
+ * @returns {boolean} whether the question is a match
+ */
+function predicateCallback(q) {
+    // Look through each widget in the question
+    for (const widget of Object.values(q.widgets)) {
+        // Skip everything that's not an interactive-graph
+        if (widget.type !== "interactive-graph") {
+            continue;
+        }
+
+        // Make sure the interactive-graph has the data we're comparing
+        if (!widget?.options?.gridStep || !widget?.options.step) {
+            continue;
+        }
+
+        // Do that actual check
+        const [xGridStep, yGridStep] = widget.options.gridStep;
+        const [xTickStep, yTickStep] = widget.options.step;
+        if (xGridStep > xTickStep || yGridStep > yTickStep) {
+            return true;
+        }
+    }
+
+    return false;
+}
```

Comment:
I like this idea! It got me scope creeping though...what if we add flags too so that people don't have to modify the file at all if the predicate they want exists! I think I'll land it and we can iterate.